content/api/rest/kom-i-gang/person.md

```diff
@@ -0,0 +1,119 @@
+
+## Autentisering med ID-porten
+
+Altinn tilbyr OIDC/OAuth2-basert autentisering og autorisasjon for eksterne integrasjoner (f.eks. sluttbrukersystemer) via ID-porten for endepunkter som krever person-autentisering. For å få tilgang til [samarbeidsportalen](https://samarbeid.digdir.no/), hvor virksomheten din kan konfigurere klienter som brukes for å aksessere Altinns API-er, ta kontakt med servicedesk@digdir.no.
+
+Altinn definerer en rekke scopes som kan brukes for å begrense tilgangen en gitt klient kan få.
+Se [liste over scopes](../../scopes#sluttbruker-api) for mer informasjon om hvordan du kan provisjonere din klient.
+
+{{% notice warning  %}}
+Altinn vil fortsette å støtte [legacy autentisering via ID-porten og cookies](../idporten-legacy/) en tid fremover, men dette mønsteret anbefales ikke for nye integrasjoner.
+{{% /notice %}}
+
+ID-porten støtter ulike flyter avhengig av implementasjon og hvordan klienten er konfigurert. Se [integrasjonsguiden](https://docs.digdir.no/oidc_guide_idporten.html) for utfyllende informasjon om hvordan du integrerer med ID-porten. 
+
+En typisk autorisasjonskode-flyt er som følger:
+
+### 1. Send sluttbruker til autorisasjonsendepunkt
+
+```
+GET https://oidc-ver2.difi.no/idporten-oidc-provider/authorize?
+  scope=altinn:instances.meta&
+  acr_values=Level3&
+  client_id=min_klient_id&
+  redirect_uri=https://eksempel.no/response& 
+  response_type=code
+```
+
+Merk at andre felter kan være påkrevd å oppgi avhengig av din implementasjon/klient. Se [integrasjonsguiden](https://docs.digdir.no/oidc_guide_idporten.html) 
+for mer informasjon.
+
+### 2. Motta autorisasjonskode på oppgitt endepunkt
+
+Etter bruker har autentisert seg (hvis ikke allerede innlogget), og har gitt tilgang din klient tilgang til scopet, blir han/hun videresendt til 
+endepunktet oppgitt i `redirect_uri` med en autorisasjonskode, f.eks. `https://eksempel.no/response?code=1JzjKYcPh4M....FMT0=`.
+
+Denne autorisasjonskode benyttes for å hente ut access_token fra ID-porten i neste steg.
+
+### 3. Hent ut access_token
+
+Avhengig av klient-type vil prosessen har noen forskjeller (se [integrasjonsguiden for mer detaljer](https://docs.digdir.no/oidc_guide_idporten.html#3-utstedelse-av-token-fra-token-endepunktet)), 
+men i hovedsak handler det om å sende  autorisasjonskoden mottatt i forrige trinn til ID-portens token-endepunkt, som da vil utstede et access_token. Responsen her vil se ut 
+noe ala dette:
+
+```
+{
+    "access_token": "eyJraWQiOiJjWmswME1rbTVIQzRnN3Z0NmNwUDVGSFp...YIcXH0AaRpxffAx7vJj6xzuIJ4C0DxnPCfRRA",
+    "token_type": "Bearer",
+    "expires_in": 3599,
+    "refresh_token": "qcCtId5...r0igT2nI",
+    "scope": "altinn:instances.meta"
+}
+```
+
+### 4. Hent ut informasjon fra Altinn
+
+Tokenet mottatt i forrige trinn benyttes mot Altinns API sammen med API-nøkkel, f.eks.:
+
+```http
```

Comment:
Jeg synes det er greit at idporten URL vises i første linje, mens vi bare viser altinn URL under Host:.

Ser helst at alle REST kommandoene mot altinn har konsekvent utlisting av headere:
Host, Accept, Content-Type, ApiKey, og ev. Authorization

(I de tilfeller hvor det kreves ?ForceEIAuthentication bør denne også vises)

Reply:
Har gått gjennom her for å være mer konsistent, og i tråd med det som faktisk sendes over wiren.